libstorage/RocksDBStorage.h

```diff
@@ -44,7 +44,7 @@ class RocksDBStorage : public Storage
 {
 public:
     typedef std::shared_ptr<RocksDBStorage> Ptr;
-
+    RocksDBStorage(bool _disableWAL = false) : m_disableWAL(_disableWAL) {}
```

Comment:
I prefer adding setXXX interfaces to modifying constructors. But I accept your modification.